spring-session/src/integration-test/java/org/springframework/session/data/mongo/MongoRepositoryJdkSerializationITests.java

```diff
@@ -15,25 +15,27 @@
  */
 package org.springframework.session.data.mongo;
 
-import java.net.UnknownHostException;
 import java.util.Map;
 
-import com.mongodb.MongoClient;
 import org.junit.Test;
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.data.mongodb.core.MongoOperations;
-import org.springframework.data.mongodb.core.MongoTemplate;
 import org.springframework.session.data.mongo.config.annotation.web.http.EnableMongoHttpSession;
+import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.context.ContextConfiguration;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
+ * Integration tests for {@link MongoOperationsSessionRepository} that use
+ * {@link JacksonMongoSessionConverter} based session serialization.
+ *
  * @author Jakub Kubrynski
+ * @author Vedran Pavic
  */
 @ContextConfiguration
+@DirtiesContext
```

Comment:
Good thing you pointed this out, ```@DirtiesContext``` can be removed now.

I've had issues similar to ones described in #396 with my initial implementation, since I had static port config for embedded MongoDB instance.

Moving to randomly assigned port fixed this.